config/output_list.json

```
@@ -1,47 +1,331 @@
+    "types": [
+      "N"
+    ]
+  },
+  "results/{sample}/cnv_sv/{sample}.manta_diploidSV.vcf.gz.tbi": {
+    "name": "_copy_manta_vcf_tbi",
+    "file": "cnv_sv/manta_run_workflow_n/{sample}/results/variants/diploidSV.vcf.gz.tbi",
+    "types": [
+      "N"
+    ]
+  },
+  "results/{sample}/cnv_sv/{sample}.tiddit.vcf.gz": {
```

Biome (2.4.15)

[error] The key results/{sample}/cnv_sv/{sample}.tiddit.vcf.gz was already declared.

(lint/suspicious/noDuplicateObjectKeys)